### PR TITLE
Add FreeRTOS Minimal and Blinky-SystemView Examples

### PR DESCRIPTION
**Pull request by e-puerto**
Add 2 Example (minimal & blinky+systemview from segger).

### CODE REVIEW

**Comment by sifivekevin** [comment]
Emmanuel, will these examples run in Freedom Studio without Segger SystemView?

**Comment by cgsfv** [comment]
I tried to create a new project in Freedom Studio using the example-freertos-minimal, and gets this result in "make all CONFIGURATION=debug" after FS have created the standalone project:
image

It doesn't seem like it builds and installs the freertos kernel. I would expect it to install the header files and the library into the highlighted directories below, just like the metal headers and libraries, or?
image

**Comment by e-puerto** [comment]
Issue with example-freertos-minimal fixed for standalone target.

File: .gitmodules [hunk 1]
```diff
@@ -112,3 +112,9 @@
 [submodule "software/example-freertos-blinky"]
 	path = software/example-freertos-blinky
 	url = https://github.com/sifive/example-freertos-blinky.git
+[submodule "software/example-freertos-minimal"]
```
Comment (bsousi5) [inline .gitmodules]: In the readme, can you add a comment indicating this example is minimal starter for their FreeRTOS application

**Pull request merged**